splinterd/tests/admin/submit.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018-2021 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Provides functionality for submitting a `CircuitManagmentPayload`
+
+use splinter::admin::client::ProposalSlice;
+use splinterd::node::Node;
+
+use crate::admin::payload::{make_circuit_disband_payload, make_circuit_proposal_vote_payload};
+use splinter::error::InternalError;
+
+// Propose a circuit disband from the specified node. The payload is returned as it is used again
+// in some tests.
+pub(super) fn propose_disband(node: &Node, circuit_id: &str) -> Result<Vec<u8>, InternalError> {
```

Comment:
These super granular wrappers will make it harder to modify/maintain the code, and my preference would be to avoid them.

Reply:
Fair, removed the commit that added this module